subprojects/docs/src/main/resources/header.html

```diff
@@ -95,92 +95,78 @@
-        </ul>
-
-        <h3 id="using-gradle">Using Gradle</h3>
-        <ul>
-            <li><a href="/userguide/installation.html">Installing Gradle</a></li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#upgrading-gradle" aria-expanded="false" aria-controls="upgrading-gradle">Upgrading Gradle</a>
+            <li><a href="/userguide/installation.html">Installation</a></li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#upgrading-gradle" aria-expanded="false" aria-controls="upgrading-gradle">Upgrading &amp; Migrating</a>
                 <ul id="upgrading-gradle">
-                    <li><a href="/userguide/upgrading_version_4.html">version 4.x to 5.0</a></li>
-                </ul>
-            </li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#migrating-to-gradle" aria-expanded="false" aria-controls="migrating-to-gradle">Migrating to Gradle</a>
-                <ul id="migrating-to-gradle">
+                    <li><a href="/userguide/upgrading_version_4.html">Upgrading from Gradle 4.x to 5.0</a></li>
                     <li><a href="https://guides.gradle.org/migrating-from-maven/">Migrating From Maven</a></li>
                 </ul>
             </li>
-            <li id="tutorials"><a href="https://guides.gradle.org/#getting-started">Getting Started Guides</a></li>
-            <li id="third-party-integration"><a href="/userguide/third_party_integration.html">Gradle &amp; Third-party Tools</a></li>
+            <li id="tutorials"><a href="https://guides.gradle.org/#getting-started">Tutorials</a></li>
         </ul>
 
         <h3 id="reference">Reference</h3>
         <ul>
-            <li><a href="https://gradle.github.io/kotlin-dsl-docs/api/" target="_blank">Kotlin DSL Reference</a></li>
-            <li><a href="/dsl/">Groovy DSL Reference</a></li>
-            <li><a href="/javadoc/">Gradle API Javadoc</a></li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#gradle-api" aria-expanded="false" aria-controls="gradle-api">Gradle API</a>
+                <ul id="gradle-api">
+                    <li><a href="/javadoc/">Javadoc</a></li>
+                    <li><a href="/dsl/">Groovy DSL</a></li>
+                    <li><a href="https://gradle.github.io/kotlin-dsl-docs/api/" target="_blank">Kotlin DSL</a></li>
+                </ul>
+            </li>
             <li><a href="/userguide/command_line_interface.html">Command-Line Interface</a></li>
             <li><a href="/userguide/plugin_reference.html">Core Plugins</a></li>
+            <li id="third-party-integration"><a href="/userguide/third_party_integration.html">Gradle &amp; Third-party Tools</a></li>
             <li><a href="https://docs.gradle.com/build-scan-plugin">Build Scan Plugin User Manual</a></li>
         </ul>
 
-        <h3 id="getting-started">Managing Gradle Projects</h3>
+        <h3 id="getting-started">Running Gradle Builds</h3>
```

Comment:
I want users to form a simple mental model of Gradle from the navigation. In particular, they should be able to work out whether they're interested how to run an existing build, or how to create and modify their own builds. I think "Running Gradle Builds" is simple to understand, whereas "Managing Gradle Projects" seems like a very broad, catch-all description.

Reply:
I really don't like either of these. 😕I think my biggest problem is the topics that go under each phrase.  For me, running would include the command line section, running Gradle in an IDE, and the info for running multi-project builds. But we also include more advanced topics like the troubleshooting and debugging info under this heading... Managing is a more inclusive word for all the other subsections, but running a Gradle build from the command line isn't really 'managing' it, to me at least.

Reply:
"Using Gradle"? :trollface: